variables.tf

```diff
@@ -25,8 +25,22 @@ variable "tags" {
   default = {}
 }
 
-variable "max_image_count" {
+variable "create_tagged_lifecycle" {
+  default = false
+}
+
+variable "create_untagged_lifecycle" {
+  default = false
```

Comment:
Use string `"false"`. 

Per documentation: 
> Therefore, using these string values rather than literal booleans is recommended when using input variables.

https://www.terraform.io/docs/configuration/variables.html